prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -0,0 +1,248 @@
+# to be able to perform the necessary tasks.  Finally the environment should be setup with "az", "kubectl", "jq",
+# and "base64" on their $PATH.
+#
+########################################################################################################################
+
+set -e
+
+# Uncomment To Debug (Verbose!)
+#set -x
+
+export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+# shellcheck disable=SC1090
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/library.sh"
+
+#
+# Global Variables
+#
+
+VARIABLES=(
+	RS_GROUP
+	REGION
+	AZURE_SUBSCRIPTION_ID
+	AZURE_SUBSCRIPTION_APP_ID
+	AZURE_SUBSCRIPTION_SECRET
+	AZURE_SUBSCRIPTION_TENANT
+	RS_GROUP
+	EVENTHUB_NAMESPACE_NAME
+)
+
+for var in "${VARIABLES[@]}"; do
+	if [ -z "${!var}" ] ; then
+		shout "ERROR: $var is not set"
+		discoverUnsetVar=true
+	fi
+done
+
+if [ "${discoverUnsetVar}" = true ] ; then
+	exit 1
+fi
+
+EVENTHUB_NAMESPACE_MIN_THROUGHPUT_UNITS=2 # Must be greater than zero and less than maximum value!
+EVENTHUB_NAMESPACE_MAX_THROUGHPUT_UNITS=4 # Must be greater than minimum value and less than 20!
+EVENTHUB_NAMESPACE_LOCATION=""
+EVENTHUB_NAMESPACE_SHARED_ACCESS_KEY="RootManageSharedAccessKey"
+K8S_SECRET_NAME="${EVENTHUB_NAMESPACE_NAME}-overrides"
+K8S_SECRET_NAMESPACE="kyma-installer"
+K8S_SECRET_USERNAME="\$ConnectionString"
+K8S_SECRET_BROKER=""
+K8S_SECRET_PASSWORD=""
+
+#
+# Utility Functions To Make The Actual Cmd Line Calls
+#
+
+createGroup() {
+	shout "Create Azure group"
+	date
+
+	az group create --name "${RS_GROUP}" --location "${REGION}"
+
+	# Wait until resource group will be visible in azure.
+	counter=0
+	until [[ $(az group exists --name "${RS_GROUP}" -o json) == true ]]; do
+		sleep 15
+		counter=$(( counter + 1 ))
+		if (( counter == 5 )); then
+			echo -e "---\nAzure resource group ${RS_GROUP} still not present after one minute wait.\n---"
+			exit 1
+		fi
+	done
+}
+
+# Create The Azure EventHubs Namespace Based On Global Configuration
+cmdCreateEventHubNamespace() {
+  az eventhubs namespace create \
+    --name "${EVENTHUB_NAMESPACE_NAME}" \
+    --resource-group "${RS_GROUP}" \
+    --location "${EVENTHUB_NAMESPACE_LOCATION}" \
+    --sku standard \
+    --enable-kafka true \
+    --enable-auto-inflate true \
+    --capacity "${EVENTHUB_NAMESPACE_MIN_THROUGHPUT_UNITS}" \
+    --maximum-throughput-units "${EVENTHUB_NAMESPACE_MAX_THROUGHPUT_UNITS}" \
+    --output none
+}
+
+# Get The Azure EventHub Namespace Authorization Keys' PrimaryConnectionString
+cmdNamespacePrimaryConnectionString() {
+  az eventhubs namespace authorization-rule keys list \
+    -o json \
+    --resource-group "${RS_GROUP}" \
+    --namespace-name "${EVENTHUB_NAMESPACE_NAME}" \
+    --name "${EVENTHUB_NAMESPACE_SHARED_ACCESS_KEY}" | \
+    jq -r ".primaryConnectionString"
+}
+
+
+# Create The EventHub Namespace Secret And Return It
+cmdCreateventHubNamespaceSecret() {
+
+  IFS="" read -r -d '' kafkaSecret << EOF
+apiVersion: v1
+stringData:
```

Comment:
stringData is wrong. See [documentation here](https://kyma-project.io/docs/components/event-bus/#details-knative-eventing-mesh-alpha-kafka) for more details.

Reply:
Fixed